modules/@apostrophecms/modal/ui/apos/components/AposModal.vue

```diff
@@ -212,36 +211,43 @@ watch(triggerFocusRefresh, (newVal) => {
   }
 });
 
-onMounted(() => {
+onMounted(async () => {
+  await nextTick();
   if (shouldTrapFocus.value) {
-    nextTick(trapFocus);
+    trapFocus();
   }
+  store.updateModalData(props.modalId, { modalEl: modalEl.value });
```

Comment:
Updating instantiated Modal in the store (from `TheAposModals`) with data only available in this component. Allows to have only one array of modals.